WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -912,6 +940,7 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
         return [
             .mentions: post.blog.isAccessibleThroughWPCom() && FeatureFlag.gutenbergMentions.enabled,
             .unsupportedBlockEditor: isUnsupportedBlockEditorEnabled,
+            .unsupportedBlockEditorSwitch: post.blog.jetpack?.isConnected ?? false,
```

Comment:
I'm wondering if something like `unsupportedBlockEditorRequiresUserInteraction` would be more descriptive?

Reply:
Also, now that we're handling Jetpack sites, should we also be updating the `isUnsupportedBlockEditorEnabled` computed property:
https://github.com/wordpress-mobile/WordPress-iOS/blob/0416d76792078e0b2a8e902e2361709844252cf2/WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift#L919-L929
to remove all the references to Jetpack?

Reply:
These names are really tricky 😬 
So, this capability is about the site being able to enable Unsupported Blocks Editor if it's disabled.
This would be the general description to no mention SSO or Jetpack. That's why it's `true` for any Jetpack site, even if SSO is already active and it doesn't require user interaction.

Maybe a better name could be `unsupportedBlockEditorSwitchable` or `...Activable`? 🤔 
I really don't like this "capability" but we need this information.

Reply:
From https://github.com/WordPress/gutenberg/pull/25238#discussion_r494792659
Maybe `unsupportedBlockEditorCanBeEnabled` could also work for the capability name 🤔 
Wdyt?

Reply:
cc @marecar3 - since we might need to change the capability name in both platforms

Reply:
> That's why it's true for any Jetpack site, even if SSO is already active and it doesn't require user interaction.

True, so `unsupportedBlockEditorRequiresUserInteraction` wouldn't work either.

> I really don't like this "capability" but we need this information.

It seems like it isn't a "capability", but I don't know where else we could put it either.

Reply:
Hah yeah, that sounds right. It's more a capability of the site that's using using Gutenberg rather than of gutenberg itself.
This feels more like a `property` of the Unsupported Block Editor capability.